### PR TITLE
fix(gas-limit): increase gas optimization constant

### LINKED ISSUE #86: increase gas optimization constant
swivel-js increases the gas estimation on a number of contract calls where `ethers`' `estimateGas` consistently underestimates

this might have several reasons, like caching and storage gas costs for tx's in the same block which might get executed on different blocks

after extended tests we found that the maximum difference between `ethers`' `estimateGas` and the actual `gasUsed` was `25971`, or about `26k`

by increasing swivel-js gas optimization to 26k we should be able to ensure that no tx created by swivel-js will fail due to gas estimation issues.

### PR DESCRIPTION
**Pull request by alexanderwende**
fixes #86

### CODE REVIEW

**Review by simonovic86 — APPROVED** [review]
lgtm

File: src/vendors/ethers/utils/gas-limit.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { ethers } from 'ethers';
 
 /**
- * This value has been calculated as the difference between estimated gas with caching on a single block
- * and actual gas without caching on multiple blocks.
+ * This value has been determined after testing all `order - fill` pairings and recording the difference
+ * between `ethers`' `estimateGas` and the tx's `gasUsed`.
  */
-const GAS_LIMIT_DELTA = ethers.BigNumber.from('15000');
+const GAS_LIMIT_DELTA = ethers.BigNumber.from('26000');
```
Comment (robrobbins) [inline src/vendors/ethers/utils/gas-limit.ts:7]: i'd just put it at 30k, but i won't object.

**Review by robrobbins — APPROVED** [review]

**Review by JTraversa — APPROVED** [review]

File: src/vendors/ethers/utils/gas-limit.ts (hunk 1 above)
Comment (alexanderwende) [reply src/vendors/ethers/utils/gas-limit.ts:7]: yeah, i was thinking about that, but that would make the gas price shown in MetaMask appear even higher (so not great from a UX perspective)... and as we had some solid tests that had a consistent upper limit of `25971`, i went with the 26k for now, also tested that, and it worked.

Comment (JTraversa) [reply src/vendors/ethers/utils/gas-limit.ts:7]: Fine either way, with this merged I'll upgrade swivel-js in my node repo and test out more order variations

**Pull request merged**